### PR TITLE
Updated anchor links

### PR DESCRIPTION
**Pull request by TimofeyDriveback**
https://app.asana.com/0/999588896812811/1124121632298568/f

### CODE REVIEW

File: _docs/digitaldata/recommendation.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ window.digitalData = {
 
 [Detailed description of the `listing` object](/digitaldata/listing)
 
-### <a name="1"></a>What to do if the list of recommendations is formed by a third-party system?
+### <a name="whatToDo"></a>What to do if the list of recommendations is formed by a third-party system?
```
Comment (bogomolov89) [inline _docs/digitaldata/recommendation.md]: whatToDo -> thirdPartySystems

File: _docs/for-developer/snippet.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Array.prototype.slice.call(arguments);c.unshift(k);a.push(c);return a}};for(b=0;
 
 >The SegmentStream snippet should be declared after the global variable `window.digitalData` and the jquery plugin.
 
-### <a name="2"></a>Installing SegmentStream on your own CDN
+### <a name="installingSegmentStreamCDN"></a>Installing SegmentStream on your own CDN
```
Comment (bogomolov89) [inline _docs/for-developer/snippet.md]: installingSegmentStreamCDN -> installingOnOwnCDN

File: _docs/integrations/facebook.md [hunk 3]
```diff
@@ -87,8 +87,7 @@ To configure the transfer of custom events, fill in 2 fields:
 
 You can add an unlimited number of custom events
 
-### <a name="5"></a>Checking the correctness of the integration setup
+### <a name="correctnessOfIntegrationSetup"></a>Checking the correctness of the integration setup
```
Comment (bogomolov89) [inline _docs/integrations/facebook.md]: correctnessOfIntegrationSetup -> checkingIntegrationCorrectness
Fix everywhere please

Comment (TimofeyDriveback) [reply _docs/integrations/facebook.md]: done

File: _docs/for-developer/snippet.md (hunk 2 above)
Comment (TimofeyDriveback) [reply _docs/for-developer/snippet.md]: done

File: _docs/digitaldata/recommendation.md (hunk 1 above)
Comment (TimofeyDriveback) [reply _docs/digitaldata/recommendation.md]: done

**Pull request merged**